src/main/java/util/XMLParser.java

```diff
@@ -0,0 +1,62 @@
+                    Element event = (Element) node;
+                    System.out.println("Event ID : " + event.getElementsByTagName("eventID").item(0).getTextContent());
+                    System.out.println("Event Type ID : " + event.getElementsByTagName("eventTypeId").item(0).getTextContent());
+                    System.out.println("Event Type Name : " + event.getElementsByTagName("eventTypeName").item(0).getTextContent());
+                    System.out.println("Venue ID : " + event.getElementsByTagName("venueId").item(0).getTextContent());
+                    System.out.println("Venue Name : " + event.getElementsByTagName("venueName").item(0).getTextContent());
+                    System.out.println("Venue Capacity : " + event.getElementsByTagName("venueCapacity").item(0).getTextContent());
+                    System.out.println("Venue Location : " + event.getElementsByTagName("venueLocation").item(0).getTextContent());
+
+                    NodeList supplierList = event.getElementsByTagName("supplier");
+                    for (int i = 0; i < supplierList.getLength(); i++) {
+                        Element supplier = (Element) supplierList.item(i);
+                        System.out.println("Supplier ID : " + supplier.getElementsByTagName("supplierId").item(0).getTextContent());
+                        System.out.println("Supplier Name : " + supplier.getElementsByTagName("supplierName").item(0).getTextContent());
+                        System.out.println("Supplier Type : " + supplier.getElementsByTagName("supplierType").item(0).getTextContent());
+                        System.out.println("Supplier Contact Name : " + supplier.getElementsByTagName("contactName").item(0).getTextContent());
+                        System.out.println("Supplier Contact Email : " + supplier.getElementsByTagName("contactEmail").item(0).getTextContent());
+                        System.out.println("Supplier Contact Phone : " + supplier.getElementsByTagName("contactPhone").item(0).getTextContent());
+                    }
+
+                    NodeList staffList = event.getElementsByTagName("staffMember");
+                    for (int i = 0; i < staffList.getLength(); i++) {
+                        Element staffMember = (Element) staffList.item(i);
+                        System.out.println("Staff ID : " + staffMember.getElementsByTagName("staffId").item(0).getTextContent());
+                        System.out.println("First Name : " + staffMember.getElementsByTagName("firstName").item(0).getTextContent());
+                        System.out.println("Last Name : " + staffMember.getElementsByTagName("lastName").item(0).getTextContent());
+                        System.out.println("Staff Member Role : " + staffMember.getElementsByTagName("role").item(0).getTextContent());
+                    }
+                }
+            }
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+}
+
```

Comment:
```suggestion
```

Reply:
It doesn't have an empty line in my local repository